pillar.example

```diff
@@ -31,3 +33,10 @@ lighttpd:
       - status
       - userdir
       - usertrack
+
+  tofs:
+    files_switch:
+      - id
+      - osfinger
+      - os
+      - os_family
```

Comment:
Would it be worth keeping the comments here?  So customising from the `template-formula`:

```
  tofs:
    # The files_switch key serves as a selector for alternative
    # directories under the formula files directory. See TOFS pattern
    # doc for more info.
    # Note: Any value not evaluated by `config.get` will be used literally.
    # This can be used to set custom paths, as many levels deep as required.
    files_switch:
      - id
      - osfinger
      - os
      - os_family
    # All aspects of path/file resolution are customisable using the options below.
    # This is unnecessary in most cases; there are sensible defaults.
    # path_prefix: template_alt
    # dirs:
    #   files: files_alt
    #   default: default_alt
    # files:
    #   lighttpd_config:
    #     - '/etc/lighttpd/lighttpd_alt.conf'
```

* Even if only the `files` commented section, since that shows which states are currently being used for `tofs` in this formula.

Reply:
done, added the comments.